blocks/carousel-slider/carousel-slider.css

```diff
@@ -0,0 +1,197 @@
+/* Carousel Slider block styles */
+.carousel-slider {
+  position: relative;
+  height: 100%;
+  width: 100%;
+}
+
+.carousel-slider .slide-container {
+  flex-direction: row;
+  position: relative;
+  height: inherit;
+  width: inherit;
+  overflow: hidden;
+}
+
+.carousel-slider .slide {
+  position: absolute;
+  height: inherit;
+  width: inherit;
+}
+
+.carousel-slider .slide > div {
+  height: inherit;
+  width: inherit;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.carousel-slider .slide > div h3 {
+  margin: 0;
+  margin-bottom: 0.5rem;
+  text-align: center;
+  font-size: 4.5rem;
+  line-height: 5rem;
+  color: #fff;
+}
+
+.who-we-are .default-content-wrapper p {
+  font-size: 1.125rem;
+  line-height: 1.75rem;
+}
+
+.carousel-slider .slide > div p {
+  font-family: "Helvetica Neue LT W05_45 Light", Helvetica, Arial, sans-serif;
+  text-align: center;
+  font-size: 1.5rem;
+  line-height: 2.25rem;
+  color: #fff;
+}
+
+.carousel-slider .btn-prev {
+  left: 0;
+}
+
+.carousel-slider .btn-prev::before {
+  content: "\e92c";
+}
+
+.carousel-slider .btn-next {
+  right: 0;
+}
+
+.carousel-slider .btn-next::before {
+  content: "\e92d";
+}
+
+.carousel-slider .btn-prev,
+.carousel-slider .btn-next {
+  position: absolute;
+  background: var(--link-color);
+  height: 50px;
+  width: 50px;
+  top: 50%;
+  margin-top: -25px;
+  cursor: pointer;
+  font-family: theme-icons;
```

Comment:
This font family is defined in lazy-styles.css.  It passes style-lint if that definition is moved within this file.  I think this can safely be ignored.

Reply:
Ok, I think this is saying to create a variable for that font, and then refer to it using the variable.
https://stylelint.io/user-guide/rules/font-family-no-missing-generic-family-keyword/

Reply:
I'll give that a try.  I also need to add mobile breakpoints.  Once I get those both done I'll resubmit the PR.